README.md

```diff
@@ -121,6 +121,19 @@ After generating a new version you should commit the changes. This will be the C
 
 Note: The Changelog will be regenerated with all tags on release on the external builder tool, but will not be checked in there. The Changelog on the github release will therefore always be correct, on the repo it might be incorrect due to missing tags when executing the `yarn release` command.
 
+## How the different .env work on deploy
+
+Each component (frontend, admin, backend and database) has its own `.env` file. When running in development you usually do not have to care about the `.env`. The defaults are set by the respective config file, found in the `src/config/` folder of each component. But if you have a local `.env`, the defaults set in the config are overwritten by the `.env`.
+
+Each component has a `.env.dist` file. This file contains all environment variables used by the component. If you want to use a local `.env`, copy the `.env.dist` and adjust the variables accordingly.
+
+Each component has a `.env.template` file. These files are very important on deploy.
```

Comment:
Does the .env.template overwrite the .env.dist?
Or in other words: which dependencies exists between all type of .env's?

Reply:
`.env.template` is solely used to generate the coresponding `.env` Ehen using the deployscripts in `deployment/bare_metal/`.
The reason is, that for the deployscript we want 1 singular `.env` (located in `deployment/bare_Metall/`), which then is spread to the corresponding module .envs.

It is to discuss if this is a good construct especially in the light of two files (.env.dist, .env.template) + code to synchronized manually